lls/nd.c

```diff
@@ -642,8 +643,8 @@ static int
 nd_pkt_valid(struct ipv6_hdr *ipv6_hdr, struct icmpv6_hdr *icmpv6_hdr,
 	uint16_t icmpv6_len)
 {
-	return ipv6_hdr->hop_limits == IPv6_DEFAULT_HOP_LIMITS &&
-		rte_be_to_cpu_16(ipv6_hdr->payload_len) == icmpv6_len &&
+	return ipv6_hdr->hop_limits == get_net_conf()->ipv6_default_hop_limits
```

Comment:
Does `RFC 4861` require `ipv6_hdr->hop_limits` to be 255, that is, this code was misusing `IPv6_DEFAULT_HOP_LIMITS`. We can hardcode 255 here because that's what the RFC requires.